cmip6_downscaling/methods/common/tasks.py

```diff
@@ -158,7 +164,7 @@ def rechunk(
     path: UPath,
     pattern: str = None,
     template: UPath = None,
-    max_mem: str = "2GB",
+    max_mem: str = "5GB",
```

Comment:
Does the new zarr encoding raise the rechunker mem limit?

Reply:
Yes. This cuts down on the number of intermediate chunks we have to generate.